extensions/binding/org.eclipse.smarthome.binding.ble.bluegiga/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,26 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: BlueGiga BLE Bridge Binding Fragment
+Bundle-SymbolicName: org.eclipse.smarthome.binding.ble.bluegiga;singleton:=true
+Bundle-Vendor: Eclipse.org/SmartHome
+Bundle-Version: 0.9.0.qualifier
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Bundle-ClassPath: .,
+ lib/com.zsmartsystems.bluetooth.bluegiga-1.0.0-SNAPSHOT.jar
+Fragment-Host: org.eclipse.smarthome.binding.ble
+Import-Package: 
+ gnu.io;version="3.12.0.OH",
```

Comment:
> + improve that EPL licensed serial driver

In general, I would love to have something like that. But I am a bit frightened of the implied efforts in that. Afair, the lib had it's only problems and bugs and especially didn't support many architectures (only [armv6hf and x86_64](https://github.com/eclipse/kura/tree/develop/target-platform/org.eclipse.soda.dk.comm-parent)?). Building the native parts of such libs can be pretty cumbersome.

> It does not use gnu.io but the javax.comm interface.

If we do not require anything special from gnu.io and we manage to have a good wrapping API from javax.comm to gnu.io, I would be fine to say that ESH code should use javax.comm for serial communication (and thus could use soda.dk), but it should also leave the option open to use RXTX as an implementation (through the wrapping API) instead.

But imho this is out of scope of this PR. To make progress here, I would suggest to move the bluegiga bundle over to openhab2-addons as it is "just another" bridge, while the official BLE support in ESH uses DBUS (btw. this bridge bundle still seems to be missing here :-))